### PR TITLE
Update dependency

### PR DESCRIPTION
**Pull request by chrushr**
### Description:

Increment atlas version from 5.3.0 to 5.3.1

Updated a function name changed in atlas https://github.com/osmlab/atlas/pull/312

### Unit Test Approach:

local build was successful

### CODE REVIEW

**Comment by chrushr** [comment]
This is currently blocked due to the function name change in https://github.com/osmlab/atlas/pull/312

File: src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public JsonObject asGeoJsonFeature(final String flagIdentifier)
         final JsonObject properties;
         if (locationItem != null)
         {
-            feature = locationItem.asGeoJsonFeature();
+            feature = locationItem.asGeoJsonGeometry();
```
Comment (yiqingj) [inline src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java]: is this a breaking change?

Comment (danielduhh) [reply src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java]: Yeah, on the Atlas side (5.3.1)

Comment (yiqingj) [reply src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java]: ahh I hate breaking changes... @matthieun  what's the reason to rename this method?

**Comment by yiqingj** [comment]
OK talked to Matthieu offline, the conclusion is: if we have to make breaking changes,  we should label the PR, and clearly mention breaking changes in release note, and jump minor version number.  Also @deprecated should be used if possible.

**Pull request merged**